### DISCUSSION TITLE
[@types/splunk-logging] Breaking change in typing that does not reflect the package

### DISCUSSION THREAD

**Discussion by Mikescops** [DefinitelyTyped/DefinitelyTyped#69360] [Issues with a @types package] [answered]
Since update v0.11.5, the `requestOptions` typings has been removed. Thus, we get the following error:

```
error TS2339: Property ‘requestOptions’ does not exist on type ‘Logger’.
```

The problem is that the `splunk-logging` has not been updated and the `requestOptions` field is still available and in use in the constructor.

https://github.com/splunk/splunk-javascript-logging/blob/cf039fbcc601bf8365b4f34397df85339ca112b4/splunklogger.js#L112

https://github.com/splunk/splunk-javascript-logging/blob/cf039fbcc601bf8365b4f34397df85339ca112b4/splunklogger.js#L300

Moreover, this was tagged as a minor change in the typing which has broken dependency bot updates on our end.

[a comment by an automated account is not shown]

**Comment by Mikescops** [comment]
The PR who is created this issue is here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69028

**Comment by ghost** [comment]
Created PR #69367 to revert this change

**Reply by Mikescops** [reply]
Thanks!

**Comment by Mikescops** [comment, marked as answer]
Reverted in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69367 --> v0.11.6
